server/src/infra/ldap_handler.rs

```diff
@@ -188,7 +188,8 @@ fn root_dse_response(base_dn: &str) -> LdapOp {
             },
             LdapPartialAttribute {
                 atype: "supportedControl".to_string(),
-                vals: vec![],
+                // Paged Results Control (RFC 2696)
+                vals: vec![b"1.2.840.113556.1.4.319".to_vec()],
```

Comment:
Hmmm... I wouldn't go as far as listing it in the supported controls. Our implementation is... rudimentary :D

Reply:
Good point, was thinking it'd be necessary, but can't really imagine it is. :)